### DISCUSSION TITLE
FOOTNOTE TITLE

### DISCUSSION THREAD

**Discussion by SuzannePAYET** [quarto-dev/quarto-cli#10792] [Q&A]
### What would you like to do?

Give feedback or suggest an improvement

### Description

Hello, I'am Suzanne PAYET.
I'm french, working for Insee.
How to do to put a title at a foonote, please ?
Thank you.

**Comment by mcanouil** [comment]
Please, use GitHub Discussion to ask question. Thank you.
See the documentation about footnotes: https://quarto.org/docs/authoring/markdown-basics.html#footnotes

Note that, it's very unclear what a "title" is for a footnote.

**Reply by SuzannePAYET** [reply]
Thank you, I know what's is my problem... this, in the document option :

language: 
  section-title-footnotes: Example Footnotes

Thank you,
By !

^^

**Reply by allefeld** [reply]
> Note that, it's very unclear what a "title" is for a footnote.

I believe @SuzannePAYET meant the title of the section containing the footnotes – how to modify it.

She (and I) found the solution in the source code of the page you pointed to:

```yaml
language:
  section-title-footnotes: Example Footnotes
```

but the page itself doesn't document this option.

Please add the documentation of this to the page?

**Reply by mcanouil** [reply]
Language is documented with all in https://quarto.org/docs/authoring/language.html

**Reply by allefeld** [reply]
The problem is that the list of options is external, in
<https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language.yml>.

So if I try to find out how to change the title of the footnote section, and search the documentation for "footnote", I will not find this list, or even the "Document Language" page which links to it.

Moreover, similar options like `abstract-title` or `toc-title` are listed e.g. under "HTML Options". If I then look there for `footnote-title` and don't find it there, I assume that no such option exists.

Wanting to change this section title is also not necessarily related to language. I wanted to change it because it bugs me that the footnotes are actually endnotes.

**Reply by mcanouil** [reply]
Then open a clear documentation issue which is not the purpose of this thread.

https://github.com/quarto-dev/quarto-cli/issues/new?template=documentation.yml

**Comment by cderv** [comment]
FWIW I created this feature request for our documentation to do better job for language option
- https://github.com/quarto-dev/quarto-cli/issues/10992

Thanks for the feedback !
